### PR TITLE
play store add apple store button added

### PR DESCRIPTION
**Pull request by dsnehasish74**
Screenshot (285)
Screenshot (286)

Issue: https://github.com/shahednasser/sbuttons/issues/355

### CODE REVIEW

File: assets/js/buttons-examples.js [hunk 1]
```diff
@@ -145,7 +145,12 @@ $(document).ready(function () {
           classes: "add-to-cart-btn",
           text: "Add To Cart",
         },
-
+        {
+          type: "apple-store",
+          label: "Apple Store Download Button",
+          classes: "applestore",
+          text: "Download",
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:152]: Change the label to "App Download Button" and add a new key `variations` where you add inside it the apple store and the play store buttons. You can follow the implementation of social media buttons for this.

File: assets/js/buttons-examples.js [hunk 2]
```diff
@@ -182,6 +187,12 @@ $(document).ready(function () {
           classes: "plus-btn",
           text: "",
         },
+        {
```
Comment (shahednasser) [inline assets/js/buttons-examples.js:190]: remove it from here and add it above in the "App Download Button"

**Review by shahednasser — CHANGES_REQUESTED** [review]
Thank you for your work @dsnehasish74 but we need them to be similar to the black buttons you see for downloading apps. Instead of all these different buttons with all the colors it will be just two buttons, one for ios and one for android. 

The implementation for this in the JS file is similar to social buttons. I have included in the review below instructions on how you can do it.

**Comment by dsnehasish74** [comment]
ok I am doing it

**Comment by dsnehasish74** [comment]
Screenshot (288)

are you asking for this type  button

**Comment by shahednasser** [comment]
Yes that's the one

**Comment by dsnehasish74** [comment]
can help me that how can I add this 2 layer text in the button ............ I mean in the upper example "Download on the " text part

**Comment by shahednasser** [comment]
> can help me that how can I add this 2 layer text in the button ............ I mean in the upper example "Download on the " text part

My suggestion is that you use ":before" for the upper text and the lower one would just be the text of the button or you can use ":after" for that as well.

**Pull request closed without merging**